nodejs/security-wg
# update: cleanup repository

This PR cleanup all the repository

* Remove unnecessary dependencies
* Remove `npm` test as we no longer uses it
* Change test suite to `node:test`
* Remove tools/reporter - no longer needed
* Update dependencies to fix vulnerabilities
